test/mock-server/jwt.ts

```diff
@@ -0,0 +1,29 @@
+import { create, getNumericDate } from './deps.ts'
+import type { Header } from './deps.ts'
+
+const EXP_IN_SECONDS = 60 * 60 // 60 minutes
+
+const payload = {
+  exp: getNumericDate(EXP_IN_SECONDS),
+  payload:
+    'TeING9+F5TB8waiN7l94SI9chwJppCYTaxTqqOdJzilmKSxh2jqsbaZ2BJoe\naR/0PhrfPUvWG2inSeLTC5M+Rg==\n',
+  uuid: 'iwokFVedG18',
+  enterprise_features: {
+    cobrand: true,
+    hideOnfidoLogo: true,
+  },
+  urls: {
+    telephony_url: 'https://localhost:8080/telephony',
+    detect_document_url: 'https://localhost:8080/sdk',
+    sync_url: 'https://sync.onfido.com',
+    hosted_sdk_url: 'https://id.onfido.com',
+    auth_url: 'https://edge.api.onfido.com',
```

Comment:
can't these also be `localhost`???

Reply:
Well the reason is 2-fold:
- For the `sync_url`, I tried to mock it but things went a bit more complicated than I thought, plus the endpoint is used for cross-device flows only, so for now we can use it as a hybrid solution.
- For the other endpoints, I can't find where we currently use them in the codebase, so I left them as is.

The purpose of this PR is to replace most of the heavy remote API requests with local requests, so I think the current state is quite balanced.